Sources/StreamChat/APIClient/Endpoints/EndpointPath.swift

```diff
@@ -125,6 +126,7 @@ enum EndpointPath: Codable {
         case let .markChannelRead(channelId): return "channels/\(channelId)/read"
         case let .markChannelUnread(channelId): return "channels/\(channelId)/unread"
         case .markAllChannelsRead: return "channels/read"
+        case .markChannelsDelivered: return "channels/delivered"
```

Comment:
Better naming would be markMessageDelivered. Although the endpoint is directly under channels, the payload has message info (and it's for a message), so it's more appropriate imo.

Reply:
Yeah I followed backend's approach. Since mark read is also related to channel. And this one is too, but feels weird.

Reply:
I'm going to stick with the current endpoint path. This is internal, so we can always change later, but at the moment, it follows the backend pattern. IMO this should live in the current user's endpoint path, since you change the messages delivered from multiple channels from the current user. So you are affecting the current user